docs/_includes/options/debug-options.md

```diff
@@ -0,0 +1,4 @@
+| Option | Long Version | Explanation                                         | Default |
+|--------|--------------|-----------------------------------------------------|---------|
+| `-d`   | `--debug`    | Print info level debug messages to the console.     | `false` |
+| `-v`   | `--verbose`  | Print detailed debug level messages to the console. | `false` |
```

Comment:
```suggestion
| `-v`   | `--verbose`  | Print trace messages to the console. | `false` |
```
not really sure if we want to change this description, up to you, but "info" and "debug" in the -d explanation is confusing since these are 2 different log levels